### PR TITLE
Batched solve

### PR DESCRIPTION
**Pull request by tobias-heinzle**
Performance improvements and some tests. 
Backward differentiation not working at the moment.

### CODE REVIEW

File: tests/test_background.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+import jax
+import jax.numpy as jnp
+
+
+from discoeb.background import evolve_background
+
+from conftest import a_RECFAST, xe_RECFAST
+
+import matplotlib.pyplot as plt
```
Comment (Findus23) [inline tests/test_background.py:11]:
```suggestion
```

I don't think it is needed for the test.

Comment (tobias-heinzle) [reply tests/test_background.py]: Thx i changed it

**Comment by ohahn** [comment]
looks great. i'll merge it.

**Comment by ohahn** [comment]
Can you open an issue regarding backward differentiability? It's actually not broken currently also in the master I think.

**Pull request merged**